static/scss/variables.scss

```diff
@@ -6,3 +6,6 @@ $red: #a31f34;
 $tile-background: #fff;
 $tiles-background: rgba(53, 53, 53, 97);
 $light-gray: #8a8b8c;
+$blue: #3dbdbf;
+$yellow: #f1ba46;
```

Comment:
Doesn't look like this is used anywhere. Also, I'm guessing this variable name is too general for its purpose.

Reply:
Yes. I will remove it.